auraescript/src/discovery.rs

```diff
@@ -29,7 +29,7 @@
 \* -------------------------------------------------------------------------- */
 
 macros::ops_generator!(
-    discovery,
+    discovery("gen/v0/discovery/discovery.ts"),
```

Comment:
it is a shame we can't generate this from the proto (version + service-name).  maybe the next iteration is a pointer to the proto and then a macro that reads the proto and parses out the services and version :D

Reply:
Ran into this with the aer PR...I wasn't able to find a suitable rust crate that parses proto files. Everything seems to rely on protoc.

Reply:
As I'm thinking about it...we don't really need a good proto parser, just something to read the proto file and extract what we want. It may be possible with the nom crate (or another one whose name I'm forgetting at the moment but is supposed to be easier to use).